spec/models/user_spec.rb

```diff
@@ -64,17 +64,17 @@
       sc.value = "true"
       sc.save!
 
-      user = FactoryBot.build(:user, kin_email: nil)
-
-      expect(user.valid?).to be false
-      expect(user.errors[:kin_email]).to eq ['Is invalid']
+      expect {
+        FactoryBot.create(:user, is_parent: false, kin_email: nil)
+      }.to raise_error(ActiveRecord::RecordInvalid)
     end
     it 'should have a mandatory kin_email field when is_parent=false upon update' do
```

Comment:
My head is going in a loop here 😵‍💫 Why is `kin_email` mandatory is `is_parent` is false?